tests/test_camera.py

```diff
@@ -0,0 +1,137 @@
+import unittest
+from unittest.mock import patch
+import sense.camera as camera
+import os
+import cv2
+
+
+class TestGetImage(unittest.TestCase):
+
+    def setUp(self) -> None:
+        cwd = os.getcwd()
+        video_path = os.path.join(cwd, 'tests/resources/test_video.mp4')
```

Comment:
The paths are used in multiple places and can be set once after the imports as a global variable.

Reply:
The paths have updated as global variables.